src/ring_theory/polynomial/chebyshev/dickson.lean

```diff
@@ -0,0 +1,126 @@
+/-
+Copyright (c) 2021 Julian Kuelshammer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Julian Kuelshammer
+-/
+
+import data.polynomial.eval
+import ring_theory.polynomial.chebyshev.defs
+
+/-!
+# Dickson polynomials
+
+The Dickson polynomials are two families of polynomials indexed by `ℕ`,
+with coefficients in a commutative ring `R` depending on an element `a∈R`.
+They are closely related to the Chebyshev polynomials in the case that `a=1`.
+When `a=0` they are just the family of monomials `x ^ n`.
+
+## Main declarations
+
+* `polynomial.dickson₁`: the Dickson polynomials of the first kind.
+* `polynomial.dickson₂`: the Dickson polynomials of the second kind.
+
+## References
+
+* Lidl, Mullen, Turnwald: "Dickson polynomials". Pitman Monographs and Surveys in Pure and Applied
+  Mathemmatics 65, Longman Group (1993)
```

Comment:
Do you mind adding this to `docs/references.bib`? Then we've been referencing the entries there like this:

https://github.com/leanprover-community/mathlib/blob/49c53d4f5ae197887ca9780f14bdf9c271b55924/src/dynamics/circle/rotation_number/translation_number.lean#L94

(Unfortunately the web docs don't process the .bib entries yet).

Reply:
I have attempted to do so. Hopefully it works like that. Thanks for the input.